std/path.d

```diff
@@ -4032,20 +4034,21 @@ version(unittest) import std.process : environment;
         if (oldHome !is null) environment["HOME"] = oldHome;
         else environment.remove("HOME");
 
-        // Test user expansion for root, no /root on Android
-        version (OSX)
-        {
-            assert(expandTilde("~root") == "/var/root", expandTilde("~root"));
-            assert(expandTilde("~root/") == "/var/root/", expandTilde("~root/"));
-        }
-        else version (Android)
+        version (Android)
         {
         }
         else
         {
-            assert(expandTilde("~root") == "/root", expandTilde("~root"));
-            assert(expandTilde("~root/") == "/root/", expandTilde("~root/"));
+            immutable user = environment.get("USER");
+            immutable tildeUser = "~" ~ user;
+            immutable echoCmd = executeShell("echo " ~ tildeUser);
+            immutable pathFromEcho = echoCmd.output.strip();
```

Comment:
Could have been one line

Reply:
I changed it slightly so I saved two lines of code but still don't have too many columns.
Putting everything into one line is way too much for my taste. It's also easier to debug with multiple lines.